src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name,
+    age,
+    sex,
+    isMarried,
+    partnerName } = person;
+  const partnerDesignation = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {person.age && (<p className="Person__age">{`I am ${age}`}</p>)}
```

Comment:
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
      
      {person.age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```
Such formatting will make your code easier to read